src/components/ChartIndexPage/chart-index.scss

```diff
@@ -1,223 +0,0 @@
-//
```

Comment:
Is this file being used by the Chart Index work that Robin was working on?? 🤔

Reply:
No I think we can remove it with no issue. It was only copied over. But I noticed that all the styles were the same so I removed them and nothing broke.